CONTRIBUTING.md

```diff
@@ -87,6 +87,12 @@ If you are not setup to detect WPCS errors, consider the following steps.
    git clone https://github.com/WordPress/WordPress-Coding-Standards.git
    ```
 
+   After this you'll need to switch to the master branch of this repo.
+   ```shell
+   cd wpcs
+   git checkout master
+   ```
+
```

Comment:
@nishantwrp Thanks for your contribution.But this is pretty obvious to start with a new repository.
@TimidRobot It is safe to close this pr for now.

Reply:
Hi, Actually wpcs is the coding standard repository not this one.It's default branch is develop. And it has some issues. Therefore we need to shift to master branch manually.

Reply:
I'd recommend you to read the PR description before commenting anything.